sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -24,6 +24,12 @@ class SystemEventNames(str, Enum):
     AcsChatThreadCreatedWithUserEventName = (
         "Microsoft.Communication.ChatThreadCreatedWithUser"
     )
+    #backward compatibility
+    AcsChatParticipantAddedToThreadEvent = "Microsoft.Communication.ChatThreadParticipantAdded"
```

Comment:
If you did miss them in the first release, don't add them. You don't have to fix something that didn't exists :)